src/Domain.LinnApps/Boms/BomChangeDetail.cs

```diff
@@ -38,6 +38,11 @@ public BomChangeDetail(BomDetail addDetail, BomDetail deleteDetail, IEnumerable<
             }
         }
 
+        public BomChangeDetail()
+        {
+            // for json deserialisation        
```

Comment:
How come we need this?

Reply:
not sure someone has changed the shape of the changerequestresource and now it 500s as it doesn't have an empty constructor to deserialise it when you do an edit